src/utils/linters.js

```diff
@@ -48,6 +48,10 @@ function isLinterConfigFromAFullyQualifiedModuleName(specifier, prefix) {
 }
 
 function resolvePresetPackage(flavour, preset, rootDir) {
+  if (flavour === 'tslint') {
```

Comment:
i needed to do this because tslint only supports node resolution. if there's a better place or way to do this, do let me know